src/conversions/chrono.rs

```diff
@@ -980,13 +980,13 @@ mod tests {
             let td = new_py_datetime_ob(py, "timedelta", (0, 3600, 0));
             let py_timedelta = new_py_datetime_ob(py, "timezone", (td,));
             // Should be equal
-            assert!(offset.as_ref(py).eq(py_timedelta).unwrap());
+            assert!(offset.bind(py).eq(py_timedelta).unwrap());
```

Comment:
It's a bit interesting to me that we didn't catch these before now.

Reply:
I think it's a result of us not testing the full feature without GIL Refs previously. As soon as I split the features, vscode started shouting at me 😅